src/main/java/com/cloudbees/jenkins/support/impl/GCLogs.java

```diff
@@ -95,7 +95,7 @@ public boolean isSelectedByDefault() {
     /**
      * Two cases:
      * <ul>
-     * <li>The file name contains <code>%t</code> or <code>%p</code> somewhere in the middle:
+     * <li>The file name contains <code>%t</code>, <code>%p</code> or <code>%n</code> somewhere in the middle:
```

Comment:
I didn't find any reference to **%n**. What does it mean? See: 

- https://docs.oracle.com/en/java/javase/11/tools/java.html#GUID-BE93ABDC-999C-4CB5-A88B-1994AAAC74D5
- https://blog.codefx.org/java/unified-logging-with-the-xlog-option/

Reply:
ditto